lms/djangoapps/course_api/blocks/transformers/block_counts.py

```diff
@@ -27,7 +27,7 @@ def collect(cls, block_structure):
         # collect basic xblock fields
         block_structure.request_xblock_fields('category')
 
-    def transform(self, usage_info, block_structure):  # pylint: disable=unused-argument
+    def transform(self, usage_info, block_structure):
```

Comment:
I believe this doesn't complain because the signature is inherited.  So it isn't a mistake to take an argument that isn't used in this particular implementation.  It only makes sense to warn about an unused argument if you can change the signature to not take the argument in the first place.